time_series_export/backend/time_series_backend.hpp

```diff
@@ -0,0 +1,171 @@
+#ifndef SERENITY_TIME_SERIES_BACKEND_HPP
+#define SERENITY_TIME_SERIES_BACKEND_HPP
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "boost/variant.hpp"
+
+#include "stout/nothing.hpp"
+#include "stout/option.hpp"
+#include "stout/try.hpp"
+
+namespace mesos {
+namespace serenity {
+
+/**
+ * Enum of data series names
+ */
+enum class Series : uint8_t {
+  SLACK_RESOURCES,
+  CPU_USAGE_SYS,
+  CPU_USAGE_USR,
+  CPU_ALLOC,
+  CYCLES,
+  INSTRUCTIONS,
+  CACHE_MISSES
+};
+
```

Comment:
There is only one new line between Series and SeriesString to emphasis their connection. I would rather keep those close. The same for Tag and TagString.